### ISSUE TITLE
Condensation Criterion in the Learned Group Convolution

### ISSUE THREAD

**Issue by showgood163** [markdtw/condensenet-tensorflow#2]
In model.py, line 131, the condensation criterion:

wis = tf.reduce_sum(wi, axis=-1)    # (w_in,)

probably should be:

wis = tf.reduce_sum(tf.abs(wi), axis=-1)    # (w_in,)

Beacuse in the original paper authors uses 'the averaged absolute value of weights'.

**Comment by markdtw** [comment]
Hi,

Sorry for the late reply. The weights are the absolute values already because of [this line](https://github.com/markdtw/condensenet-tensorflow/blob/master/model.py#L118).

Thanks!
